### PR TITLE
change map Cartography and initial zoom map

### LINKED ISSUE #438: Change Map Cartography
# Problem

We are currently using the ugly OSM cartography. 

# Proposed Solution 

Choose a better one!

### LINKED ISSUE #439: Centre Map
# Problem

Currently, ​the map defaults to a random location.

# Proposed Solution

The maps should default to organisation location.

### PR DESCRIPTION
**Pull request by ann26**
fix #438 and fix #439 

## On the certifying organisation details page:
screenshot from 2017-06-15 15-38-04

## On the training center details page:
screenshot from 2017-06-15 15-40-42

### CODE REVIEW

**Comment by ann26** [comment]
updated map view:
screenshot from 2017-06-26 22-20-04

File: django_project/certification/templates/certifying_organisation/detail.html [hunk 1]
```diff
@@ -18,29 +18,28 @@
...
-                        return e.properties.ISO2 == '{{ certifyingorganisation.country }}';
-                    });
-
-                // Put marker of the country on the map
-                var lat = geo_obj[0].properties.LAT;
-                var long = geo_obj[0].properties.LON;
-{#                marker = L.marker( [ lat,long ] ).addTo( map );#}
-            });
-
+{#            marker = L.marker( [ lat,long ] ).addTo( map );#}
```
Comment (dimasciput) [inline django_project/certification/templates/certifying_organisation/detail.html]: If you don't need this, just remove this line.

Comment (ann26) [inline django_project/certification/templates/certifying_organisation/detail.html]: ah yes, sure

**Comment by dimasciput** [comment]
@ann26 LGTM, just have one little feedback. After you address it, you can merge this.

**Pull request merged**